.gitignore

```diff
@@ -48,4 +48,7 @@ test.conf
 spark-warehouse
 *.flattened-pom.xml
 
-seatunnel-examples
\ No newline at end of file
+seatunnel-examples
+
+# vscode
+.vscode
```

Comment:
Why add it?

Reply:
to avoid uploading the my own vscode settings.